scripts/linters/custom_eslint_checks/rules/protractor-practices.js

```diff
@@ -31,27 +31,32 @@ module.exports = {
     fixable: null,
     schema: [],
     messages: {
-      disallowSleep: 'Please do not use browser.sleep() in protractor files',
-      disallowThen: 'Please do not use .then(), consider async/await instead',
```

Comment:
Avoid unwanted changes*

Reply:
Changed this because I have to Arrange message names alphabetically.